### DISCUSSION TITLE
Possible bug: All photos being sorted into incorrect date folders

### DISCUSSION THREAD

**Discussion by jackch1** [photoprism/photoprism#1965] [Q&A]
Hi

All my files have been added to photoprism via the import function, so I'm assuming should be copied/moved into the correct date folder during this process.  However this doesnt seem to be the case.

 
**Search/timeline view:** Photos in correct order
**Calendar view:** Photos in correct month
**Originals/File locations:** Seems to be random.
See the screenshot below of my May 2021 folder view:

<img width="858" alt="Screenshot of May 2021 folder" src="https://user-images.githubusercontent.com/43887297/150657744-cca4d40e-5f68-40b5-8a68-887aac06adc7.png">

Any ideas why this happend and how I can fix it?

My files are origianlly from Google Photos, I have checked the exif data and Json files and they are all correct (date/time etc).

Using PhotoPrism on Unraid Docker, latest version.
4 core i5 CPU
4GB Ram
Running on 2TB of SSD storage

**Comment by graciousgrey** [comment]
I guess the sorting of this folder is just not set to "newest first" or "oldest first". If you are in the folder (as on the screenshot) click the edit icon on top of the page, then you can change the sort order.

**Reply by jackch1** [reply]
This folder should not even contain anything that isn’t dated May 2022, changing the sort order doesn’t help

**Comment by jackch1** [comment]
I’ll report this as a bug as I can’t see any way this is functioning as expected

**Reply by lastzero** [reply]
Did you check the issue template for reporting bugs? It seems some information we need is missing. Do you mind adding it before reporting? Merci beaucoup ❤️
